pkg/server/datastore/sqlstore/sqlstore_test.go

```diff
@@ -2934,12 +2930,6 @@ func (s *PluginSuite) testListRegistrationEntries(dataConsistency datastore.Data
 					entryIDMap[entryOut.EntryId] = entryIn.EntryId
 				}
 
-				// Optionally sleep to give time for the entries to propagate to
-				// the replicas.
-				if dataConsistency == datastore.TolerateStale && s.readOnlyDelay > 0 {
-					time.Sleep(s.readOnlyDelay)
-				}
```

Comment:
This requires some bigger changes to the test to be able to also test using `Eventually`. It didn't seem to be required though, probably because we first check with `RequireCurrent` and by the time that finishes the data has replicated already.